spec/index.bs

```diff
@@ -2282,6 +2282,29 @@ dictionary GPUImageBitmapCopyView {
 
   * {{GPUImageBitmapCopyView/origin}}: If unspecified, defaults to `[0, 0]`.
 
+### {{GPUCommandEncoder/copyBufferToBuffer()|GPUCommandEncoder.copyBufferToBuffer(GPUBuffer, GPUSize64, GPUBuffer, GPUSize64, GPUSize64)}} ### {#GPUCommandEncoder-copyBufferToBuffer}
+
+<div algorithm="GPUCommandEncoder.copyBufferToBuffer(GPUBuffer, GPUSize64, GPUBuffer, GPUSize64, GPUSize64)">
+
+The <dfn method for="GPUCommandEncoder">copyBufferToBuffer(|source|, |sourceOffset|, |destination|, |destinationOffset|, |size|)</dfn> method is used to perform {{GPUBuffer}} to {{GPUBuffer}} copies.
+
+1. Ensure {{GPUCommandEncoder/copyBufferToBuffer()}} is called before {{GPUCommandEncoder/finish()}}.
+1. Ensure {{GPUCommandEncoder/copyBufferToBuffer()}} is called outside of any {{GPURenderPassEncoder}}s.
+1. Ensure {{GPUCommandEncoder/copyBufferToBuffer()}} is called outside of any {{GPUComputePassEncoder}}s.
+1. Ensure |source| is a valid {{GPUBuffer}}.
```

Comment:
Maybe |source| is not an error {{GPUBuffer}}?

Reply:
We define the term `valid` so this should be `|source| is a [=valid=] {{GPUBuffer}}.`

Reply:
Done